### DISCUSSION TITLE
Version 1.5.18 shows develop?

### DISCUSSION THREAD

**Discussion by nicedevil007** [TandoorRecipes/recipes#3219] [Q&A] [answered]
Hey guys,

I went through the update process and updated from 1.5.14 to 1.5.18. The systemscreen showed the right 1.5.14 prior the update. After the update it shows "develop", is this normal behaviour?

![image](https://github.com/user-attachments/assets/22a90e6a-626e-4ddf-a592-295bd5a5de0f)

I used the update process from this documentation page.
https://docs.tandoor.dev/install/manual/#updating

**Comment by smilerz** [comment]
you updated to the development branch, not latest.

**Reply by nicedevil007** [reply]
> you updated to the development branch, not latest.

how can I specify it? just did the `git pull` as mentioned in the docs.

**Reply by smilerz** [reply, marked as answer]
`git checkout <branch/tag>`

**Reply by nicedevil007** [reply]
mhhhh, I was able to do the checkout part but it is still on the commit with cc83 at the beginning, and at least github is telling us, that this is the latest commit version for origin/master for 1.5.18. Is it a problem that it is doing the stuff with `python3 version.py` at the end or that the development branch is now set to default on github?

![image](https://github.com/user-attachments/assets/2fb2e81e-6cdc-4fee-9faa-56eab13fca8e)

**Reply by smilerz** [reply]
master and 1.5.18 are different commits.  develop has always been the default branch.

**Reply by nicedevil007** [reply]
ok then I didn't got it :(

`git checkout develop/1.5.18` ?

**Reply by smilerz** [reply]
`git checkout 1.5.18`

**Reply by nicedevil007** [reply]
guess I got it now
![image](https://github.com/user-attachments/assets/02a2bebd-cbc5-40c2-875d-165dc9effc8a)
